jupyter_scheduler/scheduler.py

```diff
@@ -263,6 +276,14 @@ def add_outputs(self, model: DescribeJob):
         model.outputs = outputs
         model.downloaded = all(output.output_path for output in outputs)
 
+    def get_local_output_path(self, model: DescribeJob) -> str:
```

Comment:
Let's try to stick with naming directories "dir" rather than "path" to help readers understand that these are directories.